src/Swift.Runtime/src/Swift/SwiftString.cs

```diff
@@ -124,15 +113,35 @@ public SwiftString(string str)
         {
             fixed (byte* utf8BytesPtr = utf8Bytes)
             {
-                _payload = PInvoke_Create(utf8BytesPtr, utf8Bytes.Length, 1);
+                var result = PInvoke_Create(utf8BytesPtr, utf8Bytes.Length, 1);
```

Comment:
Can you document what this `1` means? Looks like a flag, but I can't find where this is.